.github/workflows/deploy.yml

```diff
@@ -1,91 +1,88 @@
+        run: npm install
+
+      - name: Build Action
+        run: npm run build
+
+      - name: Only build app
         uses: ./
         with:
+          workingDirectory: "./test/base"
           apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
-          workingDirectory: "test"
-          publish: false
-          secrets: |
-            SECRET1
-            SECRET2
-          preCommands: echo "*** pre commands ***"
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          postCommands: npx wrangler deploy --dry-run && exit 0
+
+      - name: Multiple commands
+        uses: ./
+        with:
+          workingDirectory: "./test/base"
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          commands: |
+            deployments
+            types
+
+      - name: Pre/Post commands
+        uses: ./
+        with:
+          workingDirectory: "./test/base"
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          preCommands: |
+            echo "*** pre commands ***"
+            echo "******"
           postCommands: |
             echo "*** post commands ***"
-            wrangler build
             echo "******"
-        env:
-          SECRET1: ${{ secrets.SECRET1 }}
-          SECRET2: ${{ secrets.SECRET2 }}
-  publish:
-    runs-on: ubuntu-latest
-    name: Publish app
-    steps:
-    - uses: actions/checkout@v2
-    - name: Publish app
-      uses: ./
-      with:
-        apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
-        environment: "production"
-        workingDirectory: 'test'
-  publish_legacy_credentials:
-    runs-on: ubuntu-latest
-    name: Publish app with legacy credentials
-    steps:
-    - uses: actions/checkout@v2
-    - name: Publish app
-      uses: ./
-      with:
-        apiKey: ${{ secrets.CLOUDFLARE_API_KEY }}
-        email: ${{ secrets.CLOUDFLARE_EMAIL }}
-        environment: "production"
-        workingDirectory: 'test'
-  publish_hardcoded_wrangler_version:
-    runs-on: ubuntu-latest
-    name: Publish app with hardcoded Wrangler version
-    steps:
-    - uses: actions/checkout@v2
-    - name: Publish app
-      uses: ./
-      with:
-        apiKey: ${{ secrets.CLOUDFLARE_API_KEY }}
-        email: ${{ secrets.CLOUDFLARE_EMAIL }}
-        environment: "production"
-        wranglerVersion: '1.5.0'
-        workingDirectory: 'test'
-  publish_secrets:
-    runs-on: ubuntu-latest
-    name: Publish app with secrets
-    steps:
-      - uses: actions/checkout@v2
-      - name: Publish app
+
+      - name: Environment support
         uses: ./
         with:
+          workingDirectory: "./test/environment"
           apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
-          environment: "production"
-          workingDirectory: "test"
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          environment: |
+            dev
```

Comment:
Why is `environment` a multiline string?

Reply:
It should not be, thanks for the callout